labs/src/main/scala/org/scalalabs/basic/lab03/RecursionPatternMatchingExercise.scala

```diff
@@ -29,39 +29,59 @@ object RecursionPatternMatchingExercise {
    * checkValuesIncrease(Seq(1,2,2)) == false
    */
   def checkValuesIncrease(seq: Seq[Int]): Boolean = {
-    error("fix me")
+    if (seq.length == 0) return false
+    if (seq.length == 1) return true
+    else
+      (seq(0) < seq(1)) match {
+      case true => return checkValuesIncrease(seq.tail)
+      case false => return false
```

Comment:
Hmm, doing a match on  boolean is definitely something you can do, but an `&&` might have sufficed here.